shapeAnalysisProject/dimension_reduction/PCA.py

```diff
@@ -13,4 +13,5 @@ def show_plot(similarity, filenames=[]):
         N = len(filenames)
         for i in xrange(N):
             ax.annotate(filenames[i].split('/')[1], (small[i][0], small[i][1]))
-    plt.show()
+    plt.title(title)
+    plt.show(block=value)
```

Comment:
if you are not blocking the `plt.show()` function while calling it, then you need to explicitly block it somewhere while executing. Otherwise, the plots will come and then the program will complete it's executing without closing the plot windows or it will close the plot windows before the user can check the plots.

Reply:
I have already taken care of these concerns:
1. I have assigned block = False for 1st plot
2. I have assigned block = True for 2nd plot

Result : Both the plots are visible simultaneously and they can be viewed by the user as long as the user does not close the window explicitly.